apps/cli-daemon/src/app/generators/generators.service.ts

```diff
@@ -36,23 +38,13 @@ export class GeneratorsService {
   }
 
   getAllGenerators(): GeneratorDefinition[] {
-    const schemasPath = pathResolve(
-      this.sessionService.cwd,
-      'node_modules/@schematics/angular'
-    );
```

Comment:
I think that if you'll use path.resolve like this it'll work for all platforms:
```typescript
const schemasPath = pathResolve(
       this.sessionService.cwd, 'node_modules', '@schematics', 'angular'
     );
```

Reply:
Awesome, updated